spec/integration_spec.rb

```diff
@@ -1036,6 +1036,25 @@
   end
 end
 
+context 'when a searchable attribute is not an attribute' do
+  let(:others) { Class.new(People) { def self.name = "People" } }
```

Comment:
Call it `people_class` or something like that, it will be easier to read afterwards.